static/templates/admin_user_list.handlebars

```diff
@@ -34,16 +34,13 @@
         {{/if}}
       {{/if}}
     </span>
-    {{#if is_active}}
-      <button class="button btn-primary open-user-form" title="{{t 'Edit User' }}" data-email="{{email}}">
+      <button class="button btn-primary open-user-form{{#unless is_active}} display-none{{/unless}}" title="{{t 'Edit User' }}" data-email="{{email}}">
           <i class="icon-vector-pencil"></i>
       </button>
-    {{/if}}
     <div class='admin-user-status'>
     </div>
   </td>
 </tr>
-{{#if is_active}}
 <tr class="user-name-form display-none" id="user_form_{{email}}">
```

Comment:
@jajodiaraghav I think this change belongs to the first commit.
I am not sure if this conditional should be removed. @timabbott Do you think it will be fine to remove the `if` conditional from here?